README.md

```diff
@@ -229,6 +229,29 @@ It supports the ArchEthic Cryptography rules which are:
   const archethic = require('archethic')
   const index = archethic.getTransactionIndex("00b1d3750edb9381c96b1a975a55b5b4e4fb37bfab104c10b0b6c9a00433ec4646", "https://www.archethic.net")
   // 0
+  ```
+
+    #### getLastTransaction(address, endpoint)
+  Query a node to find the last address of a chain
+  
+  - `address` Transaction address (in hexadecimal)
+  - `endpoint` Node endpoint
+
+  ```js
+  const archethic = require('archethic')
+  const index = archethic.getLastTransaction("00b1d3750edb9381c96b1a975a55b5b4e4fb37bfab104c10b0b6c9a00433ec4646", "https://www.archethic.net")
+  // 0
+  ```
+    #### getTransactionChain(address, endpoint)
```

Comment:
On this function, you should add a `page` parameter, as the results will be paginated.